src/numerical_method.py

```diff
@@ -171,8 +171,7 @@ def __init__(self) -> None:
 
 
 class PDEParams(Params):
-    def __init__(self, und_step: int, time_step: int, stock_min: int, stock_max: int, method: str) -> None:
-        # todo: to be implemented
+    def __init__(self, und_step: int = 2, time_step: float = 1/1200, stock_min: int = 0, stock_max: int = 200, method: str = 'explicit') -> None:
```

Comment:
Use enum for method.

Reply:
Is stock_max the edge of the grid? So is S within [0,200] ? We have a few underlying (MONEY_MALL) that has current spot >200. Shouldn't we use a multiplicator istead?

Reply:
This has been fixed now.